tasks/install-on-Windows.yml

```diff
@@ -50,6 +50,7 @@
     url: https://github.com/buildkite/agent/releases/download/v{{ buildkite_agent_version }}/buildkite-agent-windows-amd64-{{ buildkite_agent_version }}.zip
     dest: 'c:/windows/temp/buildkite-agent-windows-{{ buildkite_agent_platform }}-{{ buildkite_agent_version }}.zip'
     timeout: 30
+    follow_redirects: all
```

Comment:
Need an update to the `meta.yml` file to indicate minimum ansible version, and corresponding changelog entry and version bump.